routing.rst

```diff
@@ -604,74 +671,52 @@ placeholder:
     To give a ``null`` default value to any placeholder, add nothing after the
     ``?`` character (e.g. ``/blog/{page?}``).
 
-Listing all of your Routes
---------------------------
-
-As your app grows, you'll eventually have a *lot* of routes! To see them all, run:
-
-.. code-block:: terminal
-
-    $ php bin/console debug:router
-
-    ------------------------------ -------- -------------------------------------
-     Name                           Method   Path
-    ------------------------------ -------- -------------------------------------
-     app_lucky_number               ANY      /lucky/number/{max}
-     ...
-    ------------------------------ -------- -------------------------------------
-
-.. index::
-   single: Routing; Advanced example
-   single: Routing; _format parameter
-
-.. _advanced-routing-example:
+.. _routing-expression_parameters:
 
-Advanced Routing Example
-------------------------
+Expression Parameters
```

Comment:
I would not call these parameters, they are completely separated from the parameter matching and more a sub-matcher. Maybe "Matching using Expressions" or "Expression Matching"?

Reply:
This is actually a very good catch. Conditions are completely unrelated to parameters. Let me think about where to put this new section. Thanks!